pkg/fission-cli/function.go

```diff
@@ -556,6 +572,19 @@ func fnUpdate(c *cli.Context) error {
 		log.Fatal(err)
 	}
 	function.Spec.InvokeStrategy = *strategy
+
+	if c.IsSet("specializationtimeout") {
+		if c.String("executortype") != types.ExecutorTypeNewdeploy {
+			log.Fatal("specializationtimeout must be greater than or equal to 120 seconds")
```

Comment:
Sorry, I suddenly notice that the message here is wrong, please correct it.

Reply:
I should have noticed that. Thank you for pointing it out.